docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -121,7 +121,7 @@ type Asset = {
   * `launchAsset`: A special asset that is the entry point of the application code. The `fileExtension` field will be ignored for this asset and SHOULD be omitted.
   * `assets`: An array of assets used by the update bundle, such as JavaScript, pictures, and fonts. All assets (including the `launchAsset`) should be downloaded to disk before executing the the update, and a mapping of asset `key`s to locations on disk should be provided to application code.
   * Properties of each asset object:
-    * `hash`: Case-insensitive, hex-encoded SHA-256 hash of the file to guarantee integrity.
+    * `hash`: Base64Url-encoded SHA-256 hash of the file to guarantee integrity. Base64Url encoding is defined by [IETF RFC 4648](https://datatracker.ietf.org/doc/html/rfc4648#section-5).
```

Comment:
There isn't really an official standard on capitalization since base64url is always written lowercased in the RFC but looking around, Base64URL is a popular capitalization e.g. https://www.javadoc.io/static/com.nimbusds/nimbus-jose-jwt/2.14.0/com/nimbusds/jose/util/Base64URL.html

```suggestion
    * `hash`: Base64URL-encoded SHA-256 hash of the file to guarantee integrity. Base64URL encoding is defined by [IETF RFC 4648](https://datatracker.ietf.org/doc/html/rfc4648#section-5).
```